server/remote_asset/fetch_server/fetch_server.go

```diff
@@ -57,8 +57,8 @@ func NewFetchServer(env environment.Env) (*FetchServer, error) {
 		return nil, status.FailedPreconditionError("A cache is required to enable the ByteStreamServer")
 	}
 	return &FetchServer{
-		env:   env,
-		cache: cache,
+		env:      env,
+		bsClient: env.GetByteStreamClient(),
```

Comment:
nit: You could just pull this off the env - feels a bit more in line with dependency injection pattern

Reply:
Done